broker-plugins/management-http/src/main/java/org/apache/qpid/server/management/plugin/servlet/rest/RestServlet.java

```diff
@@ -437,7 +437,10 @@ public String getRequestURL()
                 {
                     if ("data".equals(part.getName()) && "application/json".equals(part.getContentType()))
                     {
-                        items = mapper.readValue(part.getInputStream(), LinkedHashMap.class);
+                        try (InputStream inputStream = part.getInputStream())
```

Comment:
The change is unnecessary as methods `ObjectMapper#readValue ` close the input stream/reader..